### DISCUSSION TITLE
error message of swanctl  on openwrt

### DISCUSSION THREAD

**Discussion by 3157696** [strongswan/strongswan#2033] [Q&A] [answered]
HI

I installed openwrt x86_84 , version  is 23.05.2 
The installed packages are：
strongswan-full（Contains strongswan-mod-wolfssl），strongswan-swanctl，kmod-crypto-gcm， strongswan-mod-gmpdh 
When starting swanctl the following error is output：
`plugin 'wolfssl' failed to load: Error relocating /usr/lib/ipsec/plugins/libstrongswan-wolfssl.so: wolfssl_ec_diffie_hellman_create: symbol not found`

Although the connection can be established, I don't know if it will have any impact

Thanks

**Comment by tobiasbrunner** [comment, marked as answer]
> Although the connection can be established, I don't know if it will have any impact

It doesn't, there are other (redundant) crypto plugins installed if you install `strongswan-full`.

**Reply by 3157696** [reply]
Because I used strongswan-minimal before, and now I migrated to swanctl, many module loading failures were displayed when starting, which I thought was abnormal.
OK now, thank you
